Js/Ass.Js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+
+        var letterGrade = $('#letter-grade').val().toUpperCase();
+        var myFavoriteThings = [];
+            myFavoriteThings.push("singing");   
+            myFavoriteThings.push("cooking"); 
+            myFavoriteThings.push("dancing"); 
+            myFavoriteThings.push("Travel"); 
+
+        //Write your decision structure between this comment...
+        if (letterGrade == "A"){
+            console.log("inside if(A)")
+            $('#decision-holder').text("Grade range for A: 90% to 100%");
+        }
+
+        else if(letterGrade == "B"){
+            $('#decision-holder').text("Grade range for B: 80% to 90%");
+        }
+
+        else if (letterGrade == "C"){
+            $('#decision-holder').text("Grade range for C: 70% to 80%");
+        }
+         else if(letterGrade == "D"){
+            $('#decision-holder').text("Grade range for D: 70% to 60%");
+         }
+         else if (letterGrade == "F"){
+            $('#decision-holder').text("Grade range for F: Less than 60%");
+         }
+         
+         else {$('#decision-holder').text("Error! Invalid value entered");}
+         
+        //...and this comment
+
+        //Now, write a for, while, OR do-while loop between this comment...
+        for (var i=0; i< myFavoriteThings.length; i++){
+            $('#loop-output').append(myFavoriteThings[i] + "<br>"); 
+        }
+
+        var counter = 0; 
+        while(counter< myFavoriteThings.length){
+            $('#each-output').append(myFavoriteThings[counter] + "<br>");
+            counter++; 
+        }
+
+        do{
+            $('#letter-grade').append(myFavoriteThings[counter] + "<br>");
+            counter++; 
+        } while (counter<myFavoriteThings);
```

Comment:
You do not need these extra loops. They will make it look like your code isn't working. Additionally - this makes your code and Clem's identical. It is very important that if you are going to work together - you declare this in your comments. Going forward - if I don't see these comments on both sides - you will both be penalized for plagiarism.